kod/object/item/passitem/numbitem/food/watrskin.kod

```diff
@@ -45,7 +45,7 @@ properties:
 
    viFilling = 6
    viNutrition = 3
-   piNumber = 20
```

Comment:
I don't know what the implications are of changing the default number of an object.  What have you found?

Reply:
It tracks the quantity of the item instance (e.g., 20 mushrooms) and acts as the default value for any instance created without a specified quantity. I’m assuming that weight and bulk are defined per single item, even for items like arrows, which are generated in multiples. If this assumption is correct, updating `piNumber` should only affect the default quantity generated when no value is specified, without causing any miscalculations.